src/app.service.spec.ts

```diff
@@ -14,11 +14,11 @@ describe('AppService', () => {
     const log: LogMessageFormat = {
         type: LogType.CPU,
         time: 1533282746739,
-        source: null,
-        detector: 'Prometheus',
-        message: "Expected CPU utilization < 80% , got 85 %for http://localhost:3000",
+        source: "http://localhost:9182/metrics",
+        detector: "http://localhost:9090/",
+        message: "CPU load is > 80%\n  VALUE = 33.8872230551391\n LABELS: map[instance:localhost:9182]",
         data: {
-            cpuUtilization: 0,
+            cpuUtilization: 33,
         },
     };
 
```

Comment:
da wir nun verschiedene mit verschiedenen log types arbeiten, sollte auch für jeden log type mindestens ein test existieren.

also:
(1) `sample-alert.json` um alters für jeden typ erweitern
(2) alerts wie gehabt in logs konvertieren
(3) logs array auf korrektheit überprüfen. hierbei am besten nicht das ganze array auf einmal (also _nicht_ so wie du es im moment überprüfts), sondern mit einem eigenen `expect` für jedes element im array. wenn dann ein log nicht korrekt konvertiert wird, kannst du leichter erkennen, welches log falsch konvertiert wurde.